lib/afc/index.js

```diff
@@ -102,6 +103,16 @@ class AfcService {
     return res.headerPayload.readUInt32LE(0);
   }
 
+  async createWriteStream (path, options) {
```

Comment:
I'd rather use another name for the `path` argument, since it may shadow the `path` module

Reply:
Makes sense